src/components/Select/Select.stories.tsx

```diff
@@ -16,7 +18,24 @@ export default meta;
 
 type Story = StoryObj<typeof meta>;
 
+function SelectWrapper({ ...arguments_ }: SelectProperties): JSX.Element {
+  const [selected, setSelected] = useState<string>('');
+
+  const onHandleChange = (
+    newValue: SelectOption | SelectOption[] | undefined
```

Comment:
No need to support array of options since this is a Single

```suggestion 
    newValue: SelectOption | undefined
```